README.md

```diff
@@ -25,15 +25,15 @@ To add items to the agenda, please join our [Open Data Hub Community group](http
 
 ## Calendar of Meetings
 To see upcoming Open Data Hub events and meetings, please add the [Open Data Hub 
-community meeting calendar](https://calendar.google.com/calendar?cid=cmVkaGF0LmNvbV92bW1pMW5tdHQyZGJjdXVqcmJ1aGo5N2pya0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
+community meeting calendar](https://meet.google.com/svm-xbwy-yui
 
-### How to Join
-1. Video  Conferencing using bluejeans: https://bluejeans.com/563715630/3948
-2. Audio Call: 
+1. Video  Conferencing using Google Meet: https://meet.google.com/svm-xbwy-yui
+2.  Audio Call: 
    - Dial one of the following numbers:
-   - 408-915-6466 (United States)
-   - For international numbers please visit [bluejeans](https://www.redhat.com/en/conference-numbers)
-   - Enter the meeting ID 563715630 #
+   - (United States) +1 575-448-4115‬ PIN: ‪387 778 528‬#
+   More phone numbers: https://tel.meet/svm-xbwy-yui?pin=9916281019745
+   Or join via SIP: sip:9916281019745@gmeet.redhat.com
```

Comment:
You'll need to indent these inline with the list to ensure they are included as part of the bulleted list.  
```suggestion
     More phone numbers: https://tel.meet/svm-xbwy-yui?pin=9916281019745
     Or join via SIP: sip:9916281019745@gmeet.redhat.com
```